src/app.config.ts

```diff
@@ -71,4 +71,10 @@ const nodeUrls = {
   TEZOS_NODE_URL,
 };
 
-export { config, nodeUrls };
+function returnNewUrl(chainName: string) {
+  const nodeUrl = process.env[`${chainName.toUpperCase()}_NODE_URL`].split(',');
```

Comment:
Imo this solution would look more robust if we use something like ETHEREUM_NODE_URL_PRIMARY, ETHEREUM_NODE_URL_SECONDARY instead of split()